lib/src/aio/local.rs

```diff
@@ -55,10 +55,12 @@ impl Backend for Local {
     fn new_thread(&self) -> io::Result<Box<dyn BackendThread>> {
         Ok(Box::new(LocalThread {
             path: self.path.clone(),
-            rand_ext: rand::thread_rng()
+            rand_ext: std::str::from_utf8(&rand::thread_rng()
```

Comment:
`.expect("must always be utf8").cloned()`

Reply:
changed